dev-docs/source/Testing/MuonAnalysis_test_guides/Muon Analysis Unscripted Testing Guide.rst

```diff
@@ -8,46 +8,85 @@ Muon Interfaces Unscripted Testing
+The testing instructions can be found at :ref:`Muon_Analysis_MUSR-ref`.
+
+You will need the following runs:
+
+- MUSR 62250-1
+- MUSR 62260
+
+Group 2: PSI data
+^^^^^^^^^^^^^^^^^
+
+This group tests bin data from the PSI facility and introduces background corrections.
+Test instructions can be found at :ref:`Muon_Analysis_PSI-ref`.
+You will need the following run:
+
+- dolly 1529
+
+Group 3: HIFI data
+^^^^^^^^^^^^^^^^^^
+
+This group looks at HIFI data and introduces multiple period data.
+Test instructions can be found at :ref:`Muon_Analysis_HIFI-ref`.
+You will need the following runs:
+
+- HIFI 134028-39
+- HIFI 84447-8
+
+Group 4: EMU data
+^^^^^^^^^^^^^^^^^
+
+This group looks at EMU data, it uses some advance fitting and plotting features.
+Test instructions for this group can be found at :ref:`Muon_Analysis_EMU-ref`.
+You will need the following runs:
+
+- EMU 51341-3
+- EMU 20889-20900
 
-Group 1: Data loading and old/new fitting UI
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Group 5: ARGUS data
+^^^^^^^^^^^^^^^^^^^
 
-Test instructions for group 1 can be found at :ref:`Muon_Analysis_TestGuide_1_General-ref`.
-These test general data loading and processing, and also switching between the old and new fitting user interfaces.
+This group looks at ARGUS data, it explores the difference between single and double pulse data.
+Testing instructions are :ref:`Muon_Analysis_ARGUS-ref`.
+You will need the following runs:
 
-Group 2: Fitting tests
-^^^^^^^^^^^^^^^^^^^^^^
+- ARGUS 71799-800
+- ARGUS 71796-7
 
-Test instructions for group 2 can be found at :ref:`Muon_Analysis_TestGuide_2_Fitting-ref`.
-These test the different types of fits. The fit should succeed in each without a crash.
-The *results* will be tested later, in the results table test section (group 3).
+Group 6: Frequequency Domain Analysis
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Group 3: Results table tests
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This group tests the Frequency domain analysis GUI.
+Test instructions for group 6 can be found at :ref:`Muon_Analysis_FDA-ref`.
+You will need the following run:
 
-Test instructions for group 3 can be found at :ref:`Muon_Analysis_TestGuide_3_Results-ref`.
-These test the generation of results tables from the fits performed in the tests in group 2.
+- MUSR 62260
 
-Group 4: Frequequency Domain Analysis tests
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Test instructions for group 4 can be found at :ref:`Muon_Analysis_TestGuide_4_FDAG-ref`.
-These test the Frequency Domain Analysis GUI and use data loaded from the tests in group 2.
+Group 7: Avoided Level Crossing (ALC)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+This group tests the Avoided Level Crossing (ALC) GUI.
+Test instructions fcan be found at :ref:`Muon_ALC-ref`.
+This requires a large number or runs (close to 100).
+Therefore, its best to do these test while connected to the data archive.
```

Comment:
I'm not keen on this, it kind of goes against the spirit of OSS

For example, ORNL (if they were concerned an unrelated change broke this) should be able to run through these dev docs to test ALC too without us having to send data around

It also works the other way, when we end up unable to review other facilities PRs since they require connection to their equivalent to a data archive 

Could we tweak the tests and / or sample data to work in a way that a user can download the data publicly and test this?

Reply:
Unfortunately the ALC just requires lots of files. Each file gives a single data point that we want to fit to. Again its from a user experiment, the only way we could store it somewhere is if someone spent a lot of time creating simulated data